src/sagemaker_xgboost_container/serving_mms.py

```diff
@@ -59,6 +61,11 @@ def _is_multi_model_endpoint():
         return False
 
 
+def _set_default_if_not_exist(sagemaker_env_var_name, default_value):
+    if not os.getenv(sagemaker_env_var_name, None):
```

Comment:
Nit: It would probably be better to check `if os.getenv(...) is None`, as it's a bit confusing to read this as is.